wp-content/themes/access/views/components/card.twig

```diff
@@ -18,6 +18,12 @@
       </a>
     </h3>
 
+    <meta itemprop="name" content="{{ post.program_name }}" />
+    <meta itemprop="datePosted" content="{{ post.post_date }}" />
```

Comment:
@theasteve Can we do `post.post_modified` instead? All of the props should be the same as in the single view.